templates/3d_compute_median_pulse_height.ipynb

```diff
@@ -12,7 +12,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    fovs = io_utils.list_folders(extracted_imgs_path)
</pre><p>If you match the FOV name format (<code>fov-</code>)that should prevent this folder, and any others, from being listed in the first place, rather than needing to search for and delete a specific subfolder</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>